documentation/book/con-user-operator-cluster-label.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// assembly-getting-started-user-operator.adoc
+
+[id='con-user-operator-cluster-label-{context}']
+
+= Identifying a Kafka cluster for user handling
+
+A `KafkaUser` resource includes a label that defines the appropriate name of the Kafka cluster (derived from the name of the `Kafka` resource) to which it belongs.
+
+[source,yaml,subs="attributes+"]
+----
+  labels:
+    strimzi.io/cluster: my-cluster
+----
+
+The label is used by the User Operator to identify the `KafkaUser` resource and create a new user.
```

Comment:
It's not just about creation but handling in general.

Reply:
The label is used by the User Operator to identify the `KafkaUser` resource and create a new user, and also in subsequent handling of the user.